### PR TITLE
Add controller CLI

### PR DESCRIPTION
**Pull request by shimks**
Docs to go with CLI creating an empty controller template
- connected to https://github.com/strongloop/loopback-next/pull/762
- connected to https://github.com/strongloop/loopback-next/issues/738

### CODE REVIEW

File: pages/en/lb4/includes/CLI-std-options.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% if include.title != "no" %}
+**Standard options**
+{% endif %}
+`-h, --help`
```
Comment (dhmlau) [inline pages/en/lb4/includes/CLI-std-options.md:4]: Do we need an extra line? 
When I was viewing it after clicking the "View" button, there's no new line before the `-h`.

**Review by dhmlau — APPROVED** [review]
Please make sure the format / spacing is correct.  Other than that, LGTM.

File: pages/en/lb4/includes/CLI-std-options.md (hunk 1 above)
Comment (shimks) [reply pages/en/lb4/includes/CLI-std-options.md:4]: I copied the file from the lb3 folder, so if we need a new line before the `-h`, we'd need to change both files. The code snippet works in loopback.io without formatting issue if that's what you're concerned about.

Comment (dhmlau) [reply pages/en/lb4/includes/CLI-std-options.md:4]: ok. If it works fine in loopback.io, then it's ok.  The "View" in github doesn't always show exactly on the web site.

**Review by bschrammIBM — COMMENTED** [review]
I am sorry, but I don't have time to parse through the history of this issue. If the rest of the team approves it, then I am fine with it.

**Pull request merged**